perpetual-haskelling-initiative.cabal

```diff
@@ -1,10 +1,10 @@
 cabal-version: 1.12
 
--- This file has been generated from package.yaml by hpack version 0.31.2.
+-- This file has been generated from package.yaml by hpack version 0.33.0.
```

Comment:
It's annoying that hpack does this (although understandable). Perhaps we should remove the `.cabal` file from version control and just let it be generated automatically whenever `stack` runs.

Reply:
Yeah, I have it removed on personal repos that have a `package.yaml`. Officially its recommended to add it to version control for people not using `stack` however that doesn't really apply here so it would probably be easier